Assets/Scripts/Game/MagicAndEffects/Effects/Restoration/HealSpellPoints.cs

```diff
@@ -0,0 +1,70 @@
+// Project:         Daggerfall Tools For Unity
+// Copyright:       Copyright (C) 2009-2018 Daggerfall Workshop
+// Web Site:        http://www.dfworkshop.net
+// License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: Hazelnut
+// Contributors:    
+// 
+// Notes:
+//
+
+using DaggerfallWorkshop.Game.Entity;
+
+namespace DaggerfallWorkshop.Game.MagicAndEffects.MagicEffects
+{
+    /// <summary>
+    /// Heal - Magicka
+    /// </summary>
+    public class HealSpellPoints : BaseEntityEffect
+    {
+        public override void SetProperties()
+        {
+            properties.Key = "Heal-SpellPoints";
+            //properties.ClassicKey = MakeClassicKey(10, 9);
+            properties.GroupName = TextManager.Instance.GetText(textDatabase, "heal");
+            properties.SubGroupName = TextManager.Instance.GetText(textDatabase, "spellPoints");
+            //properties.SpellMakerDescription = DaggerfallUnity.Instance.TextProvider.GetRSCTokens(1549);
+            //properties.SpellBookDescription = DaggerfallUnity.Instance.TextProvider.GetRSCTokens(1249);
+            properties.SupportMagnitude = true;
+            properties.AllowedTargets = EntityEffectBroker.TargetFlags_Self;
+            properties.AllowedElements = EntityEffectBroker.ElementFlags_MagicOnly;
+            properties.AllowedCraftingStations = MagicCraftingStations.PotionMaker;
+            //properties.MagicSkill = DFCareer.MagicSkills.Restoration;
+            //properties.MagnitudeCosts = MakeEffectCosts(8, 28);
+        }
```

Comment:
Please verify that I have these settings right for an effect that's only used by a potion recipe.

Reply:
This looks OK on a read. Assuming tests OK in game as well? I'll test soon.